release/ray_release/util.py

```diff
@@ -144,7 +144,7 @@ def reinstall_anyscale_dependencies() -> None:
 
     # Copy anyscale pin to requirements.txt and requirements_buildkite.txt
     subprocess.check_output(
-        "pip install -U anyscale==0.5.76",
+        "pip install -U anyscale==0.5.81",
```

Comment:
do we want to fix the anyscale version? wouldn't it be better to always install the latest anyscale version?

Reply:
We have always fixed the anyscale version - let's not change that in this PR, but definitely something we should consider overall - I'll discuss with the rest of the oss infra team